Documentation/ApiOverview/DirectoryStructure/Index.rst

```diff
@@ -96,20 +115,10 @@ So all files like CSS, JavaScript, Icons, Fonts, Images, etc. of extensions
 are not linked anymore directly to the extension folders but to the directory
 :file:`_assets/`.
 
-.. note::
-    This directory :file:`_assets/` and the related changes depend on the
-    composer-plugin `typo3/cms-composer-installers` in version 4+.
-    Previous versions of `typo3/cms-composer-installers` used the classical
-    directory structure with :file:`typo3conf/ext/` for extensions.
-
-    The Composer plugin `typo3/cms-composer-installers` in version 4+ was created
-    for TYPO3 version 12 and backported for default but **optional usage**
-    in TYPO3 version 11. Therefore the version has to be explicitely set (decreased)
-    if the classical directory structure shall be used:
-
-.. code-block:: none
-
-   "typo3/cms-composer-installers": "^3.1",
+..  note::
+    In TYPO3 v12 using the `typo3/cms-composer-installers` in version
```

Comment:
I would rephrase it, as it is installed automatically, the sentence now can be understood as the user has to/can do something.
Maybe: "TYPO3 v12 requires typo3/cms-composer-installers in version 5."